docs/DeveloperGuide.md

```diff
@@ -282,37 +282,33 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `TBM` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: Adding a Client**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User meets secures a new business deal/client
+2.  User attempts to add the all associated parties and their information into TBM.
+3.  User successfully adds the all new information into TBM.
```

Comment:
The new business deal might involve some party which the user has encountered before and thus might have an entry in TBM, but forgot. So I wrote the uc in a vague way to put in the duplicate entry uc.

Reply:
LGTM